src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,177 @@
+package Duke;
+
+import Duke.Exception.EmptyDeadlineException;
+import Duke.Exception.EmptyEventsException;
+import Duke.Exception.EmptyToDoException;
+import Duke.Exception.NullCommandException;
+import Duke.Task.Deadlines;
+import Duke.Task.Events;
+import Duke.Task.Task;
+import Duke.Task.ToDos;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+
+
+public class Duke {
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        try {
+            printFileContents("dukeData.txt");
+        } catch (FileNotFoundException e){
+            System.out.println("File not found");
+        }
+        greetLine();
+        addList();
+        exitLine();
+    }
+
+    private static final int MARK_INDEX = 5;
+    private static final int UNMARK_INDEX = 7;
+    private static final int DELETE_INDEX = 7;
+
+    public static void greetLine() {
+        System.out.println("How may I be of service today?");
+    }
+
+    public static void echo() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while (!line.equals("bye")) {
+            System.out.println(line);
+            line = in.nextLine();
+        }
+    }
+
+    public static void exitLine() {
+        System.out.println("Glad I could be of help!");
+    }
+
+    public static void addList() {
```

Comment:
Do remember to provide appropriate names to methods. The method below is not just adding a list or adding to a list.